javascript/javascript1/week4/js-week4.js

```diff
@@ -0,0 +1,111 @@
+let userName="";
+let toDoList=[];
+function getReply(command){
+    command=command.toLowerCase();
+    let resualt='command is not defined!';
+    
+    if (command.includes("hello my name is ")){
+        if(userName===''){
+            userName=command.substr(17,command.length-1);
+            if(userName===''){
+                resualt='please inter your name!';
+            }else{
+                resualt='nice to meet you '+userName;
```

Comment:
Can you see the userName in this string? I'm afraid there's something missing in order to store it and return it.